android/sdl_android/src/androidTest/java/com/smartdevicelink/test/Validator.java

```diff
@@ -1985,7 +1985,7 @@ public static boolean validateTireStatus(TireStatus item1, TireStatus item2) {
             return (item1 == null);
         }
 
-        if (item1.getPressureTellTale() != item2.getPressureTellTale()) {
```

Comment:
@kostyaBoss I would suggest reverting this change.  These are valid changes, since the `pressureTellTale` functions are being renamed per the parameter, but it is out of scope of the PR.